python/tests/res/enkf/test_site_config.py

```diff
@@ -32,14 +32,7 @@ def test_invalid_user_config(self):
     def test_init(self):
         with TestAreaContext("site_config_init_test") as work_area:
             work_area.copy_directory(self.case_directory)
-            site_config = SiteConfig()
```

Comment:
Why are you removing this test although you have not changed the site config constructor?

Reply:
This belongs to the site_config PR, will remove from this one